src/main/scala/fi/aalto/cs/apluscourses/intellij/actions/REPLAction.scala

```diff
@@ -0,0 +1,25 @@
+package fi.aalto.cs.apluscourses.intellij.actions
+
+import com.intellij.openapi.actionSystem.{AnActionEvent, CommonDataKeys}
+import com.intellij.openapi.module.{ModuleManager, ModuleUtilCore}
+import org.jetbrains.plugins.scala.console.actions.RunConsoleAction
+
+class REPLAction extends RunConsoleAction {
```

Comment:
yepp, I could not find a simple way to extend [relevant Scala class](https://github.com/JetBrains/intellij-scala/blob/ca22e710b389b0eb58cdb5b69aae62bf3273d442/scala/scala-impl/src/org/jetbrains/plugins/scala/console/actions/RunConsoleAction.scala) in Java, so, well, welcome our first it's a Scala class :rofl: 
@nikke234 must be happy!

Reply:
While I would love to have more Scala in our plugin, I'll look into doing this in java, so that our plugin is written in a single language :smile: